### PR TITLE
update docker engine

### PR DESCRIPTION
**Pull request by inductor**

### CODE REVIEW

**Comment by nohaapav** [comment]
@inductor Did you try to build swarmpit image with new version locally ? As it is failing with given error:

screen shot 2019-02-09 at 5 03 03 pm

File: Dockerfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ MAINTAINER Pavol Noha <pavol.noha@gmail.com>
 RUN apk add --update curl && \
     rm -rf /var/cache/apk/*
 
-ENV VERSION "18.03.0-ce"
+ENV VERSION "18.09.0-ce"
```
Comment (lumir-mrkva) [inline Dockerfile]: this version doesn't exists on url in next line

**Comment by inductor** [comment]
Seems like they do not have "stable" labeled version in the URL  https://download.docker.com/linux/static/stable/x86_64/

**Comment by inductor** [comment]
@lumir-mrkva and yes there is the name of the image in the URL.

**Comment by lumir-mrkva** [comment]
@inductor there is not one with "-ce", that's why the build fails

**Comment by inductor** [comment]
Oops sorry didn't notice that!

**Comment by inductor** [comment]
@lumir-mrkva does it look ok?

**Comment by lumir-mrkva** [comment]
@inductor it does, but i just have to deploy it manually to check it

**Pull request merged**